src/github/deployment.ts

```diff
@@ -80,3 +98,32 @@ export const processDeployment = async (
 		gitHubAppId
 	);
 };
+
+const persistentSuccessDeploymentStatusToDynamoDB = async (
```

Comment:
nit: I feel like "dynamodb" suffix is implementation detail, should we maybe drop it?